lib/nimble_template/version.ex

```diff
@@ -0,0 +1,31 @@
+defmodule NimbleTemplate.Version do
+  @moduledoc false
+
+  alias NimbleTemplate.Generator
+
+  def bump(new_version) do
+    current_version = Mix.Project.config()[:version]
+
+    if new_version > current_version do
```

Comment:
Even after checking the tests, I am not sure how this comparison works 😆 How can it compare strings?

Reply:
It seems like Elixir support compares the same string pattern, eg: with the same pattern `x.x.x` Elixir can compare `4.0.0` vs `4.0.1`

![image](https://user-images.githubusercontent.com/11751745/172520416-a40bf68a-0188-4adb-90b5-8d4f8144e73c.png)